sankeerth/JDBCExercises/src/main/java/BooksTable.java

```diff
@@ -0,0 +1,43 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class BooksTable {
+
+    static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    static final String url = "jdbc:mysql://localhost:3306/myDatabase";
+
+    static final String USER = "root";
+    static final String PASS = "sankeerth85";
```

Comment:
:x: look into ways to avoid pushing credentials into version control. You can you use env variables or system variables to drive them